ftime() is deprecated and causes issues for the python bindings.
From freebsd patch-shared+commonlib.c 1.2.

--- shared/commonlib.c.orig	Sun Feb  1 19:31:59 2009
+++ shared/commonlib.c	Wed May  2 14:29:13 2012
@@ -6,7 +6,7 @@
 #elif defined EnhTime
 # include <windows.h>
 #else
-# include <sys/timeb.h>
+# include <sys/time.h>
 #endif
 
 #include <stdlib.h>
@@ -844,10 +844,12 @@ double timeNow(void)
   }
   return( timeBase + (double) now.QuadPart/(double) freq.QuadPart );
 #else
-  struct timeb buf;
+  struct timeval tv;
+  struct timezone tz;
 
-  ftime(&buf);
-  return((double)buf.time+((double) buf.millitm)/1000.0);
+  gettimeofday(&tv, &tz);
+  return((double)tv.tv_sec+((double)tv.tv_usec)/1000000.0);
+
 #endif
 }
 
